tests/TestCase.php

```diff
@@ -64,6 +64,11 @@ protected function tearDown(): void
         );
     }
 
+    protected function buildKeyCache($key): string
+    {
+        return ctype_alnum($key) && mb_strlen($key, '8bit') <= 32 ? $key : md5($key);
```

Comment:
Are you sure it's needed?

Reply:
if it is better, I guarantee a unique id.